timeless/views.py

```diff
@@ -0,0 +1,33 @@
+from http import HTTPStatus
+
+from flask import views
+from werkzeug.exceptions import abort
+
+
+class CrudView(views.MethodView):
+    """View that supports generic crud operations.
+      @todo #123:30min Continue with the implementation of CrudView.
+       Implement post, put and delete methods.
+       This view should render templates specified by user.
+       Example of using CrudView:
+
+       class CommentView(CrudView):
+
+            model = Comment
+
+            templates = {
```

Comment:
@Vatavuk Templates here unnecessary, since `MethodView` is made for restful API.